autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -213,15 +234,21 @@ private void scaleDownByNodeIds(List<String> decommissionNodeIds) {
             LOGGER.info("Sending request to remove  nodeIdCount '{}', nodeId(s) '{}' from host group '{}', cluster '{}', user '{}'",
                     decommissionNodeIds.size(), decommissionNodeIds, hostGroup, cluster.getStackCrn(),
                     cluster.getClusterPertain().getUserCrn());
+            FlowIdentifier flowIdentifier = notTriggered();
             if (REGULAR.equals(scalingAdjustmentType)) {
-                cloudbreakCommunicator.decommissionInstancesForCluster(cluster, decommissionNodeIds);
+                flowIdentifier = cloudbreakCommunicator.decommissionInstancesForCluster(cluster, decommissionNodeIds);
             } else if (STOPSTART.equals(scalingAdjustmentType)) {
-                cloudbreakCommunicator.stopInstancesForCluster(cluster, decommissionNodeIds);
+                flowIdentifier = cloudbreakCommunicator.stopInstancesForCluster(cluster, decommissionNodeIds);
             }
+            String downscaleTriggerSuccessMsg = messagesService.getMessageWithArgs(AUTOSCALE_DOWNSCALE_TRIGGER_SUCCESS, decommissionNodeIds,
```

Comment:
Aren't these changes also required for 
`scaleDown(int scalingAdjustment, int totalNodes)` ?

Reply:
Updated to include `scaleDown(int scalingAdjustment, int totalNodes)` as well